src/main/java/it/pn/frontend/e2e/common/DettaglioNotificaSection.java

```diff
@@ -101,4 +106,101 @@ public void selezioneVediDettaglioButton() {
     public String getTextLinkAttestazioniOpponibili(int i) {
         return attestazioniFile.get(i).getText();
     }
+
+    public void checkTestoBoxPagamento(By xpath, Boolean isTrue) {
+        try {
+            if (isTrue){
+                getWebDriverWait(10).withMessage("campo non trovato").until(ExpectedConditions.visibilityOfElementLocated(xpath));
+            }else {
+                getWebDriverWait(10).withMessage("campo trovato").until(ExpectedConditions.invisibilityOfElementLocated(xpath));
+            }
+        } catch (TimeoutException e) {
+            logger.error("il box pagamento non é stato caricato correttamente: " + e.getMessage());
+            Assert.fail("il box pagamento non é stato caricato correttamente: " + e.getMessage());
+        }
+    }
+
+    public void checkElementVisible(By xpath) {
+        try {
+            WebElement element = getWebDriverWait(10)
+                    .withMessage("Campo non trovato")
+                    .until(ExpectedConditions.visibilityOfElementLocated(xpath));
+            if (!element.isDisplayed()) {
+                throw new TimeoutException("Il campo è stato trovato ma non è visibile.");
+            }
+        } catch (TimeoutException e) {
+            logger.error("Il box pagamento non è stato caricato correttamente: " + e.getMessage());
+            Assert.fail("Il box pagamento non è stato caricato correttamente: " + e.getMessage());
+        }
+    }
+
+    public void checkElementNotVisible(By xpath) {
+        try {
+            boolean isNotVisible = getWebDriverWait(10)
+                    .withMessage("Campo trovato")
+                    .until(ExpectedConditions.invisibilityOfElementLocated(xpath));
+            if (!isNotVisible) {
+                throw new TimeoutException("Il campo è stato trovato ed è visibile.");
+            }
+        } catch (TimeoutException e) {
+            logger.error("Il box pagamento non è stato caricato correttamente: " + e.getMessage());
+            Assert.fail("Il box pagamento non è stato caricato correttamente: " + e.getMessage());
+        }
+    }
```

Comment:
Check if this logic could work
```suggestion
 public boolean isFieldDisplayed(By xpath) {
        getWebDriverWait(10).withMessage("Campo non trovato").until(ExpectedConditions.visibilityOfElementLocated(xpath));
        return this.element(xpath).isDisplayed();
    }
```

Reply:
done